tests/layer_tests/tensorflow_tests/test_tf_ArgMinMax.py

```diff
@@ -19,7 +19,8 @@ def _prepare_input(self, inputs_info):
         input_shape = inputs_info['input']
         inputs_data = {}
         rng = np.random.default_rng()
-        inputs_data['input'] = rng.integers(-8, 8, input_shape).astype(self.input_type)
+        # Because unstable sorting is chosen by default, input tensor with unrepeated elements should be generated.
+        inputs_data['input'] = rng.choice(np.product(input_shape), size=input_shape, replace=False).astype(self.input_type)
```

Comment:
This is how the bug can be reproduced with repeating elements. This case should be supported.